### DISCUSSION TITLE
Display Option: Enable Click Action

### DISCUSSION THREAD

**Discussion by benhaynes** [directus/directus#6938] [Feature Requests]
Would be nice to enable native actions on Displays (eg: Raw and/or Formatted Value), so that actions can be taken wherever they are visible. Such as:
 
* Click to Copy Value
* Click to Open Link/Email
* Click to Toggle (boolean only, perhaps too interactive?)
* Click to Update (perhaps too interactive?)

I think Copy Value could be used anywhere a display is seen. Open Link might be a bit annoying in some places (eg: a Card title), but maybe not? Toggle/Update are pretty intrusive, and could cause more of an annoyance in places... but it depends on  how they are activated:

These actions could be on native click, show an action icon on hover, or even just have a right-click menu activated (least intrusive). The right-click (or similar) seems like it might be the best option... but you'd probably expect to "just click" a toggle on a Table layout, so maybe it's based on where it's used?

Just thinking out loud!

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
